js/rtorrent.js

```diff
@@ -694,7 +694,7 @@ rTorrentStub.prototype.getValue = function(values,i)
 		var el = value.childNodes[0];
 		while(!el.tagName)
 			el = el.childNodes[0];
-		ret = $type(el.textContent) ? $.trim(el.textContent) : 
+		ret = $type(el.textContent) ? String.prototype.trim(el.textContent) : 
```

Comment:
Try to use normal form (here and below). I.e. el.textContent.trim() istead String.prototype.trim(el.textContent).